.github/workflows/build-container.yml

```diff
@@ -70,16 +71,19 @@ jobs:
       if: steps.cache-image-restore.outputs.cache-hit != 'true'
       run: |
         export SHELL=/bin/bash
-        eval $(minikube -p minikube docker-env)
         DOCKER_IMAGE_VERSION=latest make image
         DOCKER_IMAGE_VERSION=latest DOCKER_PLUGINS_IMAGE_NAME=headlamp-plugins-test make build-plugins-container
         echo -n "verifying images:"
         docker images
+    - name: Import images to kind
+      run: |
+        export SHELL=/bin/bash
+        kind load docker-image ghcr.io/headlamp-k8s/headlamp-plugins-test:latest --name test
```

Comment:
I wonder if you tested the case where a person is only making changes to the e2e tests. In this situation the image should be cached, and not rebuilt.

Reply:
Glanced over this, thanks for the catch